dbt_project.yml

```
@@ -149,6 +149,11 @@ models:
       bnb:
         +schema: nomiswap_bnb
 
+    maverick:
+      +schema: maverick
+      ethereum:
+        +schema: ethereum
```

It seems I cannot directly commit to this PR.
Please fix this:
```suggestion
        +schema: maverick_ethereum
```